asreview/datasets.py

```diff
@@ -300,77 +292,103 @@ def find(self, dataset_name):
+            if not is_iterable(include):
+                include = [include]
+            groups = include
+        elif exclude is not None:
+            if not is_iterable(exclude):
+                exclude = [exclude]
+            groups = self.groups.copy()
+            for group_id in exclude:
+                try:
+                    groups.remove(group_id)
+                except ValueError:
+                    pass
         else:
-            group_names = group_name
+            groups = self.groups.copy()
 
         dataset_groups = get_entry_points('asreview.datasets')
 
-        dataset_list = {}
-        for group in group_names:
+        group_list = []
+        for group in groups:
             try:
-                dataset_list[group] = \
-                    dataset_groups[group].load()().list(latest_only=latest_only)
+                group_list.append(dataset_groups[group].load()())
             except Exception as err:
 
                 # don't raise error on loading entry point
                 if raise_on_error:
                     raise err
 
-        return dataset_list
+        if serialize:
+            dataset_list_ser = []
+            for data_group in group_list:
+                try:
+                    group_ser = []
+                    for dataset in data_group.datasets:
+                        group_ser.append(dataset.__dict__())
+                    dataset_list_ser.append({
+                        "group_id": data_group.group_id,
+                        "description": data_group.description,
+                        "datasets": group_ser
+                    })
+                except Exception as err:
+                    # don't raise error on loading entry point
+                    if raise_on_error:
+                        raise err
+
+            return dataset_list_ser
+
+        return group_list
 
 
 class BenchmarkDataGroup(BaseDataGroup):
     group_id = "benchmark"
     description = "A collections of labeled datasets for benchmarking."
 
     def __init__(self):
-        meta_file = "https://raw.githubusercontent.com/asreview/systematic-review-datasets/master/index.json"  # noqa
-        datasets = download_from_metadata(meta_file)
+        meta_file = "https://raw.githubusercontent.com/asreview/systematic-review-datasets/master/index_v1.json"  # noqa
+        datasets = _download_from_metadata(meta_file)
 
         super(BenchmarkDataGroup, self).__init__(
             *datasets
         )
 
 
-def download_from_metadata(url):
-    """Download metadata to dataset."""
-
-    try:
-        with urlopen(url, timeout=10) as f:
-            meta_data = json.loads(f.read().decode())
-    except URLError as e:
-        if isinstance(e.reason, socket.timeout):
-            raise Exception("Connection time out.")
-        raise e
+class NaturePublicationDataGroup(BaseDataGroup):
+    group_id = "benchmark-nature"
+    description = "Featured benchmarking datasets from the Nature publication."
```

Comment:
What is a good description (displayed in the GUI)?

Reply:
Featured datasets from Nature